website/packages/docs/src/pages/api-css.mdx

```diff
@@ -1,22 +1,9 @@
 ---
 section: 5-API
-order: 99
-name: CSS
+order: 13
+name: '[Archived] CSS'
 ---
 
 # CSS
 
-Define styles that can be statically typed and useable with other Compiled APIs.
-
-> **Opaque styles**<br/>
-> The return value of `css` at runtime is `null` and can only be used with other Compiled APIs.
-
-```jsx
-const redText = css({
-  color: 'red',
-});
-
-<div css={redText} />;
-```
-
-Read [composition](/composition) for more information around composing styles together.
+The contents of this page have been merged into the [css prop](/api-css-prop) documentation. Check it out!
```

Comment:
keeping this page just so people don't get 404s

couldn't find an easy way to set up some kind of redirect, or maybe there's a better way you can think of?